demos/notebooks/demo_caiman_cnmf_3D.ipynb

```diff
@@ -546,10 +543,7 @@
    "metadata": {},
    "source": [
     "### View the results\n",
-    "For a change we view the components as max-projections (frontal in the XY direction, sagittal in YZ direction and transverse in XZ), and we additionaly show the denoised trace\n",
-    "\n",
-    "**Note `nb_view_components_3d()` is buggy since a new Bokeh release, and we will work on a fix soon.     \n",
-    "If you know Bokeh and would like to help, please let us know!**"
+    "For a change we view the components as max-projections (frontal in the XY direction, sagittal in YZ direction and transverse in XZ), and we additionaly show the denoised trace"
```

Comment:
I know it was the original wording, but "For a change" is really weird phrasing; might be worth rephrasing it while it's being touched.